benchmark-workflow/actions/benchmark/slurm/action.yml

```diff
@@ -0,0 +1,43 @@
+name: 'Slurm Benchmark'
+description: 'Run slurm benchmark suite'
+inputs:
+  backend:
+    description: 'Backend to use'
+    required: true
+  system:
+    description: 'Name of system (e.g. GPU name)'
+    required: true
+  options:
+    description: 'Options to pass to futhark bench'
+    required: false
+    default: ''
+  slurm:
+    description: 'Options to pass to srun'
+    required: false
+    default: ''
+  gpu:
+    description: 'The GPU and numbers of GPUs to use'
+    required: false
+    default: 'none'
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/download-artifact@v3
+      with:
+        name: futhark-nightly-linux-x86_64.tar.xz
+    - name: Setup compiler
+      shell: bash
+      run: |
+        tar xvf futhark-nightly-linux-x86_64.tar.xz
+    - name: Save slurmbench.py
+      shell: bash
+      run: |
+        curl https://raw.githubusercontent.com/diku-dk/futhark-ci/slurm/slurmbench.py > slurmbench.py
+    - name: Run benchmarks
+      shell: bash
+      run: |
+        python3.9 slurmbench.py --gpu=${{inputs.gpu}} --futhark="$PWD/futhark-nightly-linux-x86_64/bin/futhark" --slurm-options=${{inputs.slurm}} --futhark-options='--ignore-files /lib/ --backend=${{inputs.backend}} --exclude no_${{inputs.system}} ${{inputs.options}}' --json=futhark-${{inputs.backend}}-${{inputs.system}}-$GITHUB_SHA.json --benchmarks=futhark-benchmarks
```

Comment:
Why are some of the options passed directly to `slurmbench.py` and others passed as a string in `--futhark-options`?  I think it would be most elegant if `slurmbench.py` simply accepted the same options as `futhark bench` (at least some of them).